src/WPF/ArcGISRuntime.WPF.Samples/Samples/Map/SearchPortalMaps/SearchPortalMaps.xaml.cs

```diff
@@ -0,0 +1,418 @@
+﻿// Copyright 2017 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Portal;
+using Esri.ArcGISRuntime.Security;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Navigation;
+
+namespace ArcGISRuntime.WPF.Samples.SearchPortalMaps
+{
+    public partial class SearchPortalMaps
+    {
+        // Variables for OAuth with default values ...
+        // URL of the server to authenticate with (ArcGIS Online)
+        private const string ArcGISOnlineUrl = "https://www.arcgis.com/sharing/rest";
```

Comment:
This is a constant variable. The naming convention for constants used by Microsoft (and enforced by StyleCop) is to use Pascal case. This isn't a hard-and-fast convention, I always used all upper for constants (ARCGIS_ONLINE_URL, e.g.), but figured we should keep with what Microsoft does (?).